Cargo.toml

```diff
@@ -33,6 +33,9 @@ scoped_threadpool = { version = "0.1", optional = true }
 tiff = { version = "0.6.0", optional = true }
 ravif = { version = "0.6.0", optional = true }
 rgb = { version = "0.8.25", optional = true }
+mp4parse = { version = "0.11.4", path = "../mp4parse/mp4parse", optional = true }
```

Comment:
A path dependency?

Reply:
This crate hasn’t been released with https://github.com/mozilla/mp4parse-rust/pull/266 yet, we’ll need one before merging this PR, hence the Draft status.

Reply:
I’ve replaced it with a git dependency instead, so that CI can pass.